server/monitor/credentials.go

```diff
@@ -0,0 +1,21 @@
+package monitor
+
+import (
+	"time"
+)
+
+func (m *Monitor) checkCredentials(force bool) {
+	if !force && (time.Now().Hour() != 0 || time.Now().Minute() != 0) {
+		return
+	}
```

Comment:
This feels a bit kludgy: would we consider using a discrete [job](https://github.com/mattermost/mattermost/blob/master/server/public/pluginapi/cluster/job.go) that runs once an hour vs. adding it to the `monitor`?

Reply:
I can move it to an independent job.

Reply:
Adding it to a job now requires extra complexity and I don't see any problem on hooking it as part of the monitoring. If you feel strong about that, we can discuss it, but I don't think it necessarily belongs to a different job.